examples/embedded/src/main/java/org/eclipse/jetty/embedded/WebSocketJsrServer.java

```diff
@@ -65,8 +62,7 @@ public static Server createServer(int port)
             }
         );
 
-        handlers.addHandler(new DefaultHandler());
-        server.setHandler(handlers);
+        server.setHandler(new HandlerList(context, new DefaultHandler()));
```

Comment:
Meh.
I like the HandlerList + addHandler view of the original.

Reply:
I think there is value in reducing 4 lines of code to 1, specially as the structure was previous spread over 15 lines.

Reply:
Since `ServletContextHandler` is a terminal `Handler`, what's the point of having a `DefaultHandler` afterwards?